src/routes/assayclassifier/testing_rule_chain.json

```diff
@@ -760,5 +766,11 @@
         "match": "is_dcwg and is_primary and dataset_type == 'MUSIC'",
         "value": "{'assaytype': 'music', 'vitessce-hints': [], 'dir-schema': 'music-v2', 'contains-pii': true, 'primary': true, 'dataset-type': 'MUSIC', 'description': 'MUSIC'}",
         "rule_description": "DCWG music"
+    },
+    {
+        "type": "match",
+        "match": "is_epic and derived_dataset_type == 'segmentation mask'",
```

Comment:
Let's change this to Segmentation Mask

Reply:
For the values, the only changes will probably be capitalization for `dataset-type` and `description`